migrator/clone/db_clone_manager_impl.go

```diff
@@ -82,8 +85,28 @@ func (d *dbCloneManagerImpl) GetCloneToMigrate() (string, string, string, error)
 		}
 
 		// If we need to migrate from rocks we need to continue processing and
-		// get the Rocks clones.  If we don't, there is no need to process Rocks
+		// get the Rocks clones.  If we don't, there is no need to process Rocks, but
+		// we will check to see if we can get rid of rocks
 		if !migrateFromRocks {
+			currentPostgresVersion := d.dbmPostgres.GetCurrentVersion()
+			// We no longer need to migrate from Rocks so we should check to see if we have upgraded
+			// to the point where we should remove it entirely
+			if rocksVersion == nil || rocksVersion.SeqNum == 0 {
+				log.Info("rocks is empty or 0, going to decommission")
+				// Obviously if the Rocks version is nil or empty we need not carry the Rocks DB forward.
+				d.decommissionRocks = true
```

Comment:
Does it also mean that there is no database to decomission?

Reply:
@c-du It means we found one but it has no version info, so it is either a fresh install and it is empty or we have already wiped it out from the migrator perspective, but central has added back in as it will until we finish clearing things back out.